environment-dev.yml

```diff
@@ -61,6 +61,7 @@ dependencies:
   - isort>=5.12
 
   - pip:
+      - git+https://github.com/data-apis/dataframe-api-compat.git@main
```

Comment:
When do we expect a new release of dataframe-api-compat containing support for Modin?

Reply:
I expect that I will have time to make a new release of this package before the new release of Modin. As a last resort, we will limit it to some working commit.